### ISSUE TITLE
Documentation for Xml lens

### ISSUE THREAD

**Issue by gasperno** [hercules-team/augeas#573]
Hi,
I find the documenation link for xml lens going to a dead page.
I am trying to access stock lenses documentation [here](http://augeas.net/stock_lenses.html).
For Xml, the documentation is linked [here](http://augeas.net/docs/references/1.4.0/lenses/files/xml-aug.html). But it's a dead link.

**Comment by raphink** [comment]
That is true, the lens has never really been properly documented. There's a few inline comments in the code, but nothing to generate online docs.
